app/containers/App.js

```diff
@@ -1,29 +1,27 @@
-import React, { Component } from 'react-native'
-import { applyMiddleware, createStore } from 'redux'
-import { Provider } from 'react-redux'
-import thunk from 'redux-thunk'
-import createLogger from 'redux-logger'
-import todoApp from '../reducers'
-import Main from '../components/Main'
+import React from 'react-native';
+import { applyMiddleware, createStore } from 'redux';
+import { Provider } from 'react-redux';
+import thunk from 'redux-thunk';
+import createLogger from 'redux-logger';
+import todoApp from '../reducers';
+import Main from '../components/Main';
 
-const logger = createLogger()
+const logger = createLogger();
 
-let createStoreWithMiddleware
+let createStoreWithMiddleware;
 
 if (__DEV__) {
-  createStoreWithMiddleware = applyMiddleware(thunk, logger)(createStore)
+  createStoreWithMiddleware = applyMiddleware(thunk, logger)(createStore);
 } else {
-  createStoreWithMiddleware = applyMiddleware(thunk)(createStore)
+  createStoreWithMiddleware = applyMiddleware(thunk)(createStore);
 }
```

Comment:
Let's more this into a file `store.js`. What do you think @alleycat-at-git @robwise ?